packages/editor/src/components/visualScript/hooks/useTemplateHandler.ts

```diff
@@ -50,7 +48,7 @@ export const useTemplateHandler = ({
 
   const createGraphTemplate = (nodes: Node[], edges: Edge[]): GraphTemplate => ({
     id: uuidv4(),
-    name: uniqueId('New template '),
+    name: 'New template ' + Math.random().toString(36).slice(-6),
```

Comment:
i think we should `uuidv4` instead

Reply:
In both cases there is a chance we will encounter the same id again, it won't break anything but the probability is low.

I'm not sure how uuidv4 works so I choose random instead

Reply:
Well, we are using uuids everywhere (even in the database as primary keys)
With uuid it can be just a shorter line `'New template ' + uuidv4()`

Reply:
we will get a giant uuid following the template, here we are keeping it short to 6 alphanumericals only